.rubocop.yml

```diff
@@ -1,6 +1,12 @@
 Metrics/LineLength:
   Enabled: false
 
+Metrics/MethodLength:
```

Comment:
Would prefer not to add more rules to make this PR pass -- however, I'll address that in another issue.  The error that is probably shown is not even related to what you added.